applications/crossbar/src/modules/cb_transactions.erl

```diff
@@ -77,6 +80,8 @@ allowed_methods() ->
 
 allowed_methods(?CURRENT_BALANCE) ->
     [?HTTP_GET];
+allowed_methods(?CREDIT) ->
+    [?HTTP_PUT];
 allowed_methods(?DEBIT) ->
     [?HTTP_DELETE];
```

Comment:
I, actually do not care which verbs to use, but it looks like PUT is the better choice for both cases.
I think PUT is better because it reflects the point of creating of transaction - debit or credit.

DELETE, in its turn, reflects an attempt to delete already created transaction. And at this moment I believe we've better avoid this feature until it is widely discussed and proven as a must.    

As of me, this PR is an attempt to understand whether core developers are open at least to touch this area. After we'll find out current position, we'll define our TODO/Feature list as discussed at 2600hz-dev list.

At the same time, since we've already got Darren's comments regarding possible billing improvements/development I'm thinking  very positive about both ways - step by step PRs or separate addon development (crossbar module/bookkeeper/etc..), because at the end of the day it should be converted into different flavors billing plugins ... dreams, dreams ... :)